handlers/handlers.go

```diff
@@ -83,21 +82,14 @@ func removeNonProtectedCookies(w http.ResponseWriter, req *http.Request) {
 				Value:    "",
 				Path:     "/",
 				Expires:  time.Unix(0, 0),
+				//MaxAge:   0,
```

Comment:
Does this need to be commented out? Consider removal if not needed.

Reply:
face palm that was stupid of me I commented out last minute to test then didn't uncomment it out :D Thanks for spotting